src/tokens/solana.tokenlist.json

```diff
@@ -91313,6 +91313,21 @@
         "website": "https://www.nftabs.io"
       }
     },
+    {
+      "chainId": 101,
+      "address": "7aENbEssDsZZ7T1HP7LLgoHJF8q4JDKfxArkmdLReUZc",
+      "symbol": "UGA",
+      "name": "BAPC Native Token",
+      "decimals": 0,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/<7aENbEssDsZZ7T1HP7LLgoHJF8q4JDKfxArkmdLReUZc/logo.png",
```

Comment:
you have a typo in the image:
you have <7aENbEssDsZZ7T1HP7LLgoHJF8q4JDKfxArkmdLReUZc/logo.png",
should be 7aENbEssDsZZ7T1HP7LLgoHJF8q4JDKfxArkmdLReUZc/logo.png